src/OpenTelemetry.ResourceDetectors.Azure/OpenTelemetry.ResourceDetectors.Azure.csproj

```diff
@@ -10,6 +10,7 @@
 
     <ItemGroup>
         <PackageReference Include="OpenTelemetry" Version="$(OpenTelemetryCoreLatestVersion)" />
+        <PackageReference Include="System.Text.Json" Version="7.0.2" />
```

Comment:
Do we need this package?
If it is needed for .NET Framework, shall we add it only for it?

Reply:
we target netstandard2.0 only at the moment so this is needed.